tests/entrypoints/openai/test_mp_crash.py

```diff
@@ -1,33 +1,21 @@
-from typing import Any
-
 import pytest
 
-from vllm.engine.async_llm_engine import AsyncLLMEngine
 from vllm.entrypoints.openai.api_server import build_async_engine_client
 from vllm.entrypoints.openai.cli_args import make_arg_parser
 from vllm.utils import FlexibleArgumentParser
 
 
-def crashing_from_engine_args(
-    cls,
-    engine_args: Any = None,
-    start_engine_loop: Any = None,
-    usage_context: Any = None,
-    stat_loggers: Any = None,
-) -> "AsyncLLMEngine":
-    raise Exception("foo")
```

Comment:
this works in main because the engine process is a fork of the api server. when we isolate them, this function is not present in the engine process.